### PR TITLE
Update languages

### PR DESCRIPTION
**Pull request by hellc**

### CODE REVIEW

File: Localization/de.lproj/Localizable.strings [hunk 1]
```diff
@@ -40,30 +42,30 @@
 "setup.welcome.createButton" = "Erstellen Sie eine neue Wallet";
 "setup.welcome.createDescription" = "Erstellen Sie eine neue Wallet auf Ihrem iOS-Gerät.";
 "setup.welcome.restoreButton" = "Stellen Sie Ihre Wallet wieder her";
-"setup.welcome.restoreDescription" = "iOS-wallet verloren? Stellen Sie es hier wieder her.";
+"setup.welcome.restoreDescription" = "Haben Sie Ihre iOS-Wallet verloren? Stellen Sie Ihre Wallet hier wieder her.";
```
Comment (SwenVanZanten) [inline Localization/de.lproj/Localizable.strings]: This text is waaaayyy to long

File: Localization/de.lproj/Localizable.strings [hunk 2]
```diff
@@ -40,30 +42,30 @@
...
 
 /*Pin unlock*/
-"setup.pinCode.title" = "PIN eingeben";
+"setup.pinCode.title" = "PIN zum Entsperren eingeben";
```
Comment (SwenVanZanten) [inline Localization/de.lproj/Localizable.strings]: Too long

File: Localization/de.lproj/Localizable.strings [hunk 3]
```diff
@@ -40,30 +42,30 @@
...
 
 /*Set wallet pin*/
-"setup.pinCode.setPinTitle" = "Wallet PIN einstellen";
-"setup.pinCode.setPinDesc" = "Dein Wallet PIN wird dazu verwendet, um die App zu öffnen und XVG mit ihr zu senden. Wenn du den PIN vergisst, kannst du nicht mehr auf dein Wallet zugreifen.";
+"setup.pinCode.setPinTitle" = "Wallet PIN festlegen";
+"setup.pinCode.setPinDesc" = "Ihre Wallet PIN benötigen Sie um auf Ihre Wallet zuzugreifen und um XVG von Ihrer Wallet zu versenden. Falls Sie Ihre Wallet PIN vergessen wird es Ihnen nicht mehr möglich sein auf Ihre Wallet zuzugreiffen.";
 
 /*Re enter wallet pin*/
-"setup.pinCode.reenterPinTitle" = "Geben Sie die PIN erneut";
+"setup.pinCode.reenterPinTitle" = "Geben Sie die PIN erneut ein";
```
Comment (SwenVanZanten) [inline Localization/de.lproj/Localizable.strings]: Too long

File: Localization/de.lproj/Localizable.strings [hunk 4]
```diff
@@ -80,17 +82,17 @@
 "setup.finish.switch3" = "Ich habe die Nutzungsbedingungen gelesen, verstanden und bin damit einverstanden.";
 "setup.finish.termsButton" = "Nutzungsbedingungen";
 "setup.finish.createButton" = "Wallet erstellen";
-"setup.finish.openButton" = "öffne dein wallet";
-"setup.finish.desc" = "Vervollständigung deines Wallets...";
-"setup.finish.congrats" = "Dein Wallet ist bereit! Glückwunsch!";
+"setup.finish.openButton" = "Öffnen Sie Ihre Wallet";
```
Comment (SwenVanZanten) [inline Localization/de.lproj/Localizable.strings]: Too long

File: Localization/de.lproj/Localizable.strings [hunk 5]
```diff
@@ -112,9 +114,9 @@
 "restoreGuide.restore.title" = "Wallet wiederherstellen";
 "restoreGuide.restore.desc" = "Auf dem nächsten Bildschirm müssen Sie den Seed Phrase Ihres Wallets in genau der gleichen Reihenfolge wie gewünscht ausfüllen.";
 "restoreGuide.restore.notice" = "Bitte stellen Sie sicher, dass Sie während dieses Vorgangs nicht beobachtet werden.";
-"restoreGuide.restore.restoreButton" = "Wiederherstellung";
+"restoreGuide.restore.restoreButton" = "Starten der Wiederherstellung";
```
Comment (SwenVanZanten) [inline Localization/de.lproj/Localizable.strings]: Too long

File: Localization/de.lproj/Localizable.strings [hunk 6]
```diff
@@ -133,14 +135,14 @@
...
+"send.noBalanceDesc" = "Sie haben keine XVG zum Versenden. Bitte erhöhen Sie Ihr Guthaben.";
 
 /*QR-code*/
-"send.qrcode.title" = "Scanne einen XVG QR Code";
+"send.qrcode.title" = "XVG QR Code scannen";
 "send.qrcode.closeButton" = "Schließen";
 
 /*Select recipient*/
-"send.selectRecipient.title" = "Wähle den Empfänger";
+"send.selectRecipient.title" = "Wählen Sie einen Empfänger aus";
```
Comment (SwenVanZanten) [inline Localization/de.lproj/Localizable.strings]: Too long

File: Localization/de.lproj/Localizable.strings [hunk 7]
```diff
@@ -189,11 +191,11 @@
 /*Addresses*/
 "settings.addresses.title" = "Adressen";
 "settings.addresses.descLabel" = "Jedes Verge Wallet kann Milliarden von Adressen aus deiner 12-Wort-Wiederherstellungsphrase generieren.  Eine neue Adresse wird automatisch generiert und bei jedem Zahlungseingang angezeigt.";
-"settings.addresses.scanButton" = "Adressen scannen um balance";
+"settings.addresses.scanButton" = "Adresse scannen um den Kontostand anzuzeigen";
```
Comment (SwenVanZanten) [inline Localization/de.lproj/Localizable.strings]: Too long

File: Localization/es.lproj/Localizable.strings [hunk 8]
```diff
@@ -43,10 +45,10 @@
 "setup.welcome.restoreDescription" = "¿Ha perdido su monedero iOS? Restáurelo aquí.";
 
 /*Pin unlock*/
-"setup.pinCode.title" = "Ingrese PIN";
+"setup.pinCode.title" = "Ingrese PIN para desbloquear";
```
Comment (SwenVanZanten) [inline Localization/es.lproj/Localizable.strings]: Too long

File: Localization/es.lproj/Localizable.strings [hunk 9]
```diff
@@ -43,10 +45,10 @@
...
 
 /*Set wallet pin*/
-"setup.pinCode.setPinTitle" = "Establezca un PIN";
+"setup.pinCode.setPinTitle" = "Establezca un PIN para el monedero";
```
Comment (SwenVanZanten) [inline Localization/es.lproj/Localizable.strings]: Too long

File: Localization/es.lproj/Localizable.strings [hunk 10]
```diff
@@ -189,7 +191,7 @@
 /*Addresses*/
 "settings.addresses.title" = "Direcciones";
 "settings.addresses.descLabel" = "Cada monedero de Verge puede generar billones de direcciónes desde su frase de contraseña de 12 palabras. Una nueva dirección es automáticamente generada y mostrada cada vez que recibe un pago.";
-"settings.addresses.scanButton" = "Escanear para saldo";
+"settings.addresses.scanButton" = "Escanear direcciones para el saldo";
```
Comment (SwenVanZanten) [inline Localization/es.lproj/Localizable.strings]: Too long

File: Localization/it.lproj/Localizable.strings [hunk 11]
```diff
@@ -189,7 +191,7 @@
 /*Addresses*/
 "settings.addresses.title" = "Indirizzi";
 "settings.addresses.descLabel" = "Ogni portafogli Verge può generare milliardi di indirizzi dalla tua chiave privata. Un nuovo indirizzo è generato e mostrato automaticamente ogni volta che ricevi un pagamento";
-"settings.addresses.scanButton" = "Scansiona indirizzi per saldo";
+"settings.addresses.scanButton" = "Scansiona gli indirizzi per il saldo";
```
Comment (SwenVanZanten) [inline Localization/it.lproj/Localizable.strings]: Too long

File: Localization/pt-PT.lproj/Localizable.strings [hunk 12]
```diff
@@ -46,7 +48,7 @@
 "setup.pinCode.title" = "Insira o PIN";
 
 /*Set wallet pin*/
-"setup.pinCode.setPinTitle" = "Defina o PIN";
+"setup.pinCode.setPinTitle" = "Defina o PIN da carteira";
```
Comment (SwenVanZanten) [inline Localization/pt-PT.lproj/Localizable.strings]: Too long

File: Localization/pt-PT.lproj/Localizable.strings [hunk 13]
```diff
@@ -189,7 +191,7 @@
 /*Addresses*/
 "settings.addresses.title" = "Endereços";
 "settings.addresses.descLabel" = "Cada carteira de verge pode gerar milhares de milhão de endereços a partir da sua frase de recuperação de 12 palavras. Um novo endereço é gerado automaticamente e exibido sempre que recebe um pagamento.";
-"settings.addresses.scanButton" = "Verificando endereços para saldo";
+"settings.addresses.scanButton" = "Verificando endereços para encontrar saldo";
```
Comment (SwenVanZanten) [inline Localization/pt-PT.lproj/Localizable.strings]: Too long

File: Localization/tr.lproj/Localizable.strings [hunk 14]
```diff
@@ -46,11 +48,11 @@
 "setup.pinCode.title" = "Kilidi açmak için PIN gir";
 
 /*Set wallet pin*/
-"setup.pinCode.setPinTitle" = "Cüzdanın PIN belirle";
-"setup.pinCode.setPinDesc" = "Cüzdan PIN’in uygulamayı açmak için kullanılır ve bununla XVG gönderilebilirsin.Eğer PIN’i kaybedersen cüzdanına erişemezsin.";
+"setup.pinCode.setPinTitle" = "Cüzdanın için PIN belirle";
```
Comment (SwenVanZanten) [inline Localization/tr.lproj/Localizable.strings]: Too long

File: Localization/de.lproj/Localizable.strings (hunk 1 above)
Comment (hellc) [reply Localization/de.lproj/Localizable.strings]: We had the same issues with Japanese I fixed before the commit. 
What TO DO: I could bring only a new keys translations but I couldn't fix issues like your example, cause I don't know the context in those languages.

**Comment by hellc** [comment]
We had the same issues with Japanese I fixed before the commit. 
Solution: I could bring only a new keys translations but I couldn't fix issues like your example, cause I don't know the context in those languages.

We need to contact with the contributors in "TOO LONG" cases or update UI to Fit It in.

**Pull request merged**